### DISCUSSION TITLE
Proper way to remove workspace under apps/packages?

### DISCUSSION THREAD

**Discussion by khewdev** [vercel/turborepo#5894] [Help] [answered]
Is there any documentation discussing about how to remove workspace in Turborepo?

**Comment by anthonyshew** [comment, marked as answer]
I'm not sure what this question means. If you're describing removing a package from your repository, you would delete it and then run the install command for your package manager to make sure it's removed from the lockfile. (You'd, of course, also need to remove it from any other packages where you've got it installed.)
